azurerm/resource_arm_monitor_action_group.go

```diff
@@ -66,6 +67,60 @@ func resourceArmMonitorActionGroup() *schema.Resource {
 				},
 			},
```

Comment:
could we add all of these new blocks [to the documentation](https://github.com/terraform-providers/terraform-provider-azurerm/blame/master/website/docs/r/monitor_action_group.html.markdown#L59)?

Reply:
I think we're all set here.